buddyboss/buddyboss-platform
# [CRITICAL] Absence of ADMIN and MODERATOR users in the group users management page!

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2470 .

### How to test the changes in this Pull Request:

1. Create a new group and add Admins, Moderators to this group
2. Create a new profile type and enable this option **Hide all members of this type from Members Directory**
3. Assign this profile type to the Group Admins and Moderators
4. Go to the website and open this Group > Manage > Members and can see all members

### Proof Screenshots or Video
https://vimeo.com/592688065/ca78f535ef


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### BuddyBoss API PR
https://github.com/buddyboss/buddyboss-platform-api/pull/339
